### PR TITLE
[updated link] Updated Docker link on README

### PR DESCRIPTION
**Pull request by tabithahsia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 Docker Community Edition (CE) >= 17.06
 Docker Compose >= 1.18
 
-Follow the [Docker installation instructions](https://www.docker.com/products/overview) for your OS.
+Follow the [Docker installation instructions](https://www.docker.com/get-docker#/overview) for your OS.
```
Comment (richardxia) [inline README.md]: Thanks for the suggestion, but maybe this link is even more direct: https://www.docker.com/community-edition#/download

It seems that Docker is trying to push their Enterprise Edition product more, so it's a bit more difficult to navigate to the free Community Edition.

Comment (tabithahsia) [reply README.md]: okiedoke

**Comment by richardxia** [comment]
I believe the tests are failing because you submitted a PR from a fork under your account. For security reasons, Travis CI will not be able to use the secret keys that we've added to the main fork on other forks, because otherwise, arbitrary people would be able to view the secret keys by messing with `.travis.yml` to print them out or something. This means that the the Travis build on this branch will not be able to connect to our SauceLabs account, and the TestCafe tests will not be able to run.

Normally, I'd ask that you resubmit the PR but from a branch that is pushed to the main repo and not a personal fork, but since this is just a documentation change, I'll just merge it in now. We should have better documentation on this, so I can work on updating the CONTRIBUTING.md guide to include instructions on our branching/Pull Request process.

**Comment by richardxia** [comment]
Thanks for the submission!

**Pull request merged**